src/main/java/com/hotels/hde/quibble/Action.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2015-2018 Expedia Inc.
+ * Copyright (C) 2015-2019 Expedia, Inc.
```

Comment:
All the license headers get updated as a side effect of modifying the project, they're all the same - updating the year and adding a "," (which according to the legal department is more correct).